app/tests/spec/lib/metrics.js

```diff
@@ -552,6 +552,16 @@ function (chai, $, p, Metrics, AuthErrors, Environment, sinon, _, WindowMock, Te
       });
     });
 
+    describe('logScreenEvent', function () {
+      beforeEach(function () {
```

Comment:
@philbooth - if I use `before` instead of `beforeEach`, `before` is run before the outer `beforeEach` that creates the dependencies, and `metrics` is undefined. This was also a problem for router.js and refresh-observer.js. Am I doing it wrong?